cli/util/uploader.go

```diff
@@ -82,7 +88,11 @@ func uploadBin(ctx *cli.Context) error {
 	fmt.Fprintln(ctx.App.Writer, "Chain block height:", currentBlockHeight)
 
 	signer := user.NewAutoIDSignerRFC6979(acc.PrivateKey().PrivateKey)
-	p, err := pool.New(pool.NewFlatNodeParams(rpcNeoFS), signer, pool.DefaultOptions())
+
+	params := pool.DefaultOptions()
+	params.SetNodeDialTimeout(defaultDialTimeout)
+	params.SetNodeStreamTimeout(defaultStreamTimeout)
```

Comment:
I still think that the one who must be blamed is the stream timeout because search operation holds underlying stream open for a long time. Let's change dial timeout to 1 minute and also set `NodeRequestTimeout` to 1 minute. I'm OK with NodeStreamTimeout of 10 minutes. Let's see if these new values are OK.